content/developer/howtos/localization.rst

```diff
@@ -0,0 +1,389 @@
+    :condition: odoo_dir_in_path
+    :language: xml
+    :start-at: tax_report
+    :end-before: account_tax_report_gstrpt_g3
+
+
+
+Fiscal positions
+----------------
+
+.. seealso::
+   - :ref:`Fiscal Position References <reference/account_fiscal_position>`
+   - :doc:`/applications/finance/accounting/taxation/taxes/fiscal_positions`
+
+Specify fiscal positions in `data/account_fiscal_position_template_data.xml` file.
+
+.. example::
+  `addons/l10n_es/data/account_fiscal_position_template_data.xml <{GITHUB_PATH}/addons/l10n_es/data/account_fiscal_position_template_data.xml>`__
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_es/data/account_fiscal_position_template_data.xml
+    :condition: odoo_dir_in_path
+    :language: xml
+    :start-at: fp_nacional
+    :end-before: fp_intra
+
+Final Steps
+===========
+
+The last step when installing a localization module is to try applying its chart of accounts to the current company (if it does not already have one).
+File `data/account_chart_template_try_loading.xml` is responsible for that.
+
+.. example::
+  `addons/l10n_ch/data/account_chart_template_data.xml <{GITHUB_PATH}/addons/l10n_ch/data/account_chart_template_data.xml>`__
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ch/data/account_chart_template_data.xml
+    :condition: odoo_dir_in_path
+    :language: xml
+    :start-at: <function
+    :end-at: </function>
+
+Finally, you may add a demo company, so the localization can be easily tested in demo mode.
+
+.. example::
+  `addons/l10n_ch/demo/demo_company.xml <{GITHUB_PATH}/addons/l10n_ch/demo/demo_company.xml>`__
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ch/demo/demo_company.xml
+    :condition: odoo_dir_in_path
+    :language: xml
+    :start-after: <odoo>
+    :end-before: </odoo>
+
+Accounting reports
+==================
+
+.. raw:: html
+
+   <div><span class="badge" style="background-color:#AD5E99">Enterprise feature</span><div>
+
+.. seealso::
+  :doc:`/applications/finance/accounting/reporting/overview`
+
+Accounting reports should be added via a separate module **l10n_XX_reports** that should go to the `enterprise repository <https://github.com/odoo/enterprise>`__.
+
+Basic `__manifest__.py` file for such a module looks as following:
+
+
+.. code-block:: py
+
+    {
+        "name": "COUNTRY - Accounting Reports",
+        "category": "Accounting/Localizations/Reporting",
+        "version": "1.0.0",
+        "license": "OEEL-1",
+        "depends": [
+            "l10n_XX", "account_reports"
+        ],
+        "data": [
+            "data/account_financial_html_report_data.xml",
+        ],
+        "auto_install": True,
+    }
+
+
+Functional overview of financial reports is here: :doc:`/applications/finance/accounting/reporting/overview/main_reports`.
+
+Some good examples:
+
+* `l10n_ch_reports/data/account_financial_html_report_data.xml <https://github.com/odoo/enterprise/blob/15.0/l10n_ch_reports/data/account_financial_html_report_data.xml>`__
```

Comment:
Wouldn't that make sense to have a replacement for enterprise URLs too?

Reply:
Done